samples/WebApp/Code.gs

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
- Could we use all lowercase file names?
- Could we rename `Code.gs` to something more descriptive?

Reply:
This repo hasn't been updated to the newer file naming conventions yet, but even then our decision was to stick with `code.js` for the entry point file.